.gitignore

```diff
@@ -40,3 +40,7 @@ lib/
 
 # cache folder
 .cache
+
+# Folder created by FileSystemAdapter
+/files
```

Comment:
Without '/' git ignores 'src/Adapters/Files/' folder too. I added this so git ignores only the first-level 'files' folder